__tests__/AppContainer.spec.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { shallow } from 'enzyme'
+import { AppContainer } from "../src/AppContainer";
+
+describe('AppContainer rendering', () => {
+    it('renders correctly', () => {
+        const component = shallow(<AppContainer/>);
+        expect(component).toMatchSnapshot();
+    });
+
+    it('renders correctly', () => {
+        const component = shallow(<AppContainer
+        productSelected = { true }
```

Comment:
Could you skip passing truthy values? Just `<AppContainer productSelected />`

Reply:
fixed